Automattic/jetpack
# Fix interstitial for Social

## Proposed changes:

* Fix interstitial for Jetpack Social, display the missing sidebar image (that gives more context about what Social is)

<img width="1127" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/90ebad20-4ec9-4694-84d0-3025ed5912d4">


### Other information:


## Jetpack product discussion


pbNhbs-9bI-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Spin up a JN site with Jetpack Beta pointing to this branch
* Visit `/wp-admin/admin.php?page=my-jetpack`
* Confirm that the interstitial now has the illustration, and it matches the screenshot above
